<!-- -->
<launch>

  <arg name="map" default="fme_open2" />

  <rosparam command="delete" ns="/$(optenv ROBOT tibi)/move_base" />

  <group ns="$(optenv ROBOT tibi)"> 
<!-- definida en el .bashrc, si quiero que use las dos diferentes, he de quitar la definicion que tengo en el bashrc-->

    <!--launch-prefix="xterm -e ddd -args" -->
  <!--output="screen"-->
    <node pkg ="move_base"
          type="move_base"
          name="move_base">
      <remap from="/$(optenv ROBOT tibi)/cmd_vel"
              to="/$(optenv ROBOT tibi)/segway/cmd_vel" />
      <remap from="/$(optenv ROBOT tibi)/odom"
              to="/$(optenv ROBOT tibi)/segway/odom" />
      <remap from="/$(optenv ROBOT tibi)/front_scan"
              to="/$(optenv ROBOT tibi)/sensors/front_laser_scan" />
      <remap from="/$(optenv ROBOT tibi)/rear_scan"
              to="/$(optenv ROBOT tibi)/sensors/rear_laser_scan" />
      <remap from="/$(optenv ROBOT tibi)/tracks"
               to="/$(optenv ROBOT tibi)/mht/tracks" />
      <rosparam file="$(find iri_akp_local_planner_companion)/config/$(optenv ROBOT tibi)/move_base_params.yaml"      command="load" />
      <rosparam file="$(find iri_akp_local_planner_companion)/config/$(optenv ROBOT tibi)/costmap_common_params.yaml" command="load" ns="global_costmap" />
      <rosparam file="$(find iri_akp_local_planner_companion)/config/$(optenv ROBOT tibi)/costmap_common_params.yaml" command="load" ns="local_costmap" />
      <rosparam file="$(find iri_akp_local_planner_companion)/config/$(optenv ROBOT tibi)/local_costmap_params.yaml"  command="load" />
      <rosparam file="$(find iri_akp_local_planner_companion)/config/$(optenv ROBOT tibi)/global_costmap_params.yaml" command="load" />
      <rosparam file="$(find iri_akp_local_planner_companion)/config/$(optenv ROBOT tibi)/akp_local_planner_params.yaml" command="load" />
      <!--<param name="AkpLocalPlanner/force_map_path"       value="$(find iri_akp_local_planner)/maps/$(arg map)_forces.txt" />-->
      <param name="AkpLocalPlanner/destination_map_path" value="$(find iri_akp_local_planner_companion)/maps/fme_open_destinations_sim.txt" />
      <param name="AkpLocalPlanner/robot" value="$(optenv ROBOT tibi)"/>
    </node>

  </group>

</launch>
